ui/components/multichain/network-list-menu/network-list-menu.tsx

```diff
@@ -604,7 +604,7 @@ export const NetworkListMenu = ({ onClose }: NetworkListMenuProps) => {
                   padding={4}
                   marginTop={2}
                   gap={4}
-                  backgroundColor={BackgroundColor.backgroundMuted}
```

Comment:
This is a Banner inside a network list. I feel like this should remain muted?

Reply:
Good suggestion. Updated https://github.com/MetaMask/metamask-extension/pull/33764/commits/e1e76b7cda36e18c5353e5f1a7bf147c293fbb12